### PR TITLE
chore/SfFooter

### PR DESCRIPTION
**Pull request by przspa**
# Related issue

# Scope of work

remove v-model from SfFooter

# Screenshots of visual changes

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/organisms/SfFooter/SfFooter.js [hunk 1]
```diff
@@ -25,23 +17,23 @@ export default {
...
         this.$nextTick(() => {
           if (mobile) {
-            opened = [];
+            open = [];
           } else {
-            opened = [...this.items];
+            open = [...this.items];
           }
-          this.$emit("change", opened);
+          this.open = open;
```
Comment (EduardDopler) [inline packages/vue/src/components/organisms/SfFooter/SfFooter.js]:
```suggestion
          this.nextTick(() => (this.open = mobile ? [] : [...this.items]));
```

File: packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.js [hunk 2]
```diff
@@ -11,9 +11,9 @@ export default {
     }
   },
   computed: {
-    opened() {
+    open() {
       const title = this.title;
-      return this.$parent.opened.includes(title);
+      return this.$parent.open.includes(title);
```
Comment (EduardDopler) [inline packages/vue/src/components/organisms/SfFooter/_internal/SfFooterColumn.js]:
```suggestion
      return this.$parent.open.includes(this.title);
```

File: packages/vue/src/components/organisms/SfFooter/SfFooter.js (hunk 1 above)
Comment (przspa) [reply packages/vue/src/components/organisms/SfFooter/SfFooter.js]: awesome, thanks 🚀

**Comment by przspa** [comment]
@EduardDopler thanks for your CR

**Pull request merged**